openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -21,6 +21,7 @@
 
 from ccx_keys.locator import CCXLocator
 
+THUMBNAIL_DIMENSIONS = (375, 200)
```

Comment:
375 x 200 is the default demo course image, and was a number that @talbs suggested might be reasonable as a default. At a later time, we can add resampled versions for all of those, but I figured that this was a reasonable (and simple) first step.